test/http.spec.js

```diff
@@ -150,4 +152,30 @@ describe('http', function () {
 
     await expect(drain(HTTP.ndjson(res.body))).to.eventually.be.rejectedWith(/aborted/)
   })
+
+  it('progress events', async () => {
+    let upload = 0
+    const body = new Uint8Array(1000000 / 2)
+    const request = await HTTP.post(`${process.env.ECHO_SERVER}/echo`, {
+      body,
+      onUploadProgress: (progress) => {
+        expect(progress).to.have.property('lengthComputable').to.be.a('boolean')
+        expect(progress).to.have.property('total', body.byteLength)
+        expect(progress).to.have.property('loaded').to.be.greaterThan(0)
```

Comment:
Minor nit, a bit more readable:

```suggestion
        expect(progress).to.have.property('loaded').that.is.greaterThan(0)
```